bodo/pandas/vendor/duckdb/src/optimizer/remove_unused_columns.cpp

```diff
@@ -446,7 +704,9 @@ void BaseColumnPruner::AddBinding(BoundColumnRefExpression &col) {
 	auto entry = column_references.find(col.binding);
 	if (entry == column_references.end()) {
 		// column not referenced yet - add a binding to it entirely
-		column_references[col.binding].bindings.push_back(col);
+		ReferencedColumn column;
+		column.bindings.push_back(col);
+		column_references.insert(make_pair(col.binding, std::move(column)));
```

Comment:
Why this change? Was just for debugging and a no-op now?

Reply:
Not strictly necessary but didn't like how it was inconsistent with the above routine.